third_party/terraform/tests/resource_compute_per_instance_config_test.go.erb

```diff
@@ -22,7 +22,7 @@ func TestAccComputePerInstanceConfig_statefulBasic(t *testing.T) {
 		"config_name3" : fmt.Sprintf("instance-%s", randString(t, 10)),
 		"config_name4" : fmt.Sprintf("instance-%s", randString(t, 10)),
 	}
-	igmId := fmt.Sprintf("projects/%s/zones/%s/instanceGroupManagers/igm-%s",
+	igmId := fmt.Sprintf("projects/%s/zones/%s/instanceGroupManagers/tf-test-igm-%s",
```

Comment:
Suggestion: put the name inside of the context map so that you don't have to construct it here and in testAccComputePerInstanceConfig_igm()